apps/vr-tests/src/stories/Accordion.stories.tsx

```diff
@@ -0,0 +1,149 @@
+      {story()}
+    </Screener>
+  ))
+  .addDecorator(FluentProviderDecorator)
+  .addStory('size', () => (
+    <Accordion index={[0, 1, 2, 3]}>
+      <AccordionItem>
+        <AccordionHeader size="small">Small</AccordionHeader>
+        <AccordionPanel>Small Panel</AccordionPanel>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionHeader size="medium">Medium</AccordionHeader>
+        <AccordionPanel>Small Panel</AccordionPanel>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionHeader size="large">Large</AccordionHeader>
+        <AccordionPanel>Small Panel</AccordionPanel>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionHeader size="extra-large">Extra Large</AccordionHeader>
+        <AccordionPanel>Small Panel</AccordionPanel>
+      </AccordionItem>
+    </Accordion>
+  ))
+  .addStory('expandIconPosition="end"', () => (
+    <Accordion expandIconPosition="end" index={[0]}>
+      <AccordionItem>
+        <AccordionHeader>Opened</AccordionHeader>
+        <AccordionPanel>Visible Panel</AccordionPanel>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionHeader>Closed</AccordionHeader>
+        <AccordionPanel>Hidden Panel</AccordionPanel>
+      </AccordionItem>
+    </Accordion>
+  ))
+  .addStory('expandIcon="<Icon/>"', () => (
+    <Accordion index={[]}>
+      <AccordionItem>
+        <AccordionHeader expandIcon={<CircleRingIcon />} expandIconPosition="start">
+          Expand Icon Start
+        </AccordionHeader>
+        <AccordionPanel>Hidden Panel</AccordionPanel>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionHeader expandIcon={<CircleRingIcon />} expandIconPosition="end">
+          Expand Icon End
+        </AccordionHeader>
+        <AccordionPanel>Hidden Panel</AccordionPanel>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionHeader inline expandIcon={<CircleRingIcon />} expandIconPosition="end">
+          Expand Icon Inline End
+        </AccordionHeader>
+        <AccordionPanel>Hidden Panel</AccordionPanel>
+      </AccordionItem>
+    </Accordion>
+  ))
+  .addStory('icon="<Icon/>"', () => (
+    <Accordion index={[]}>
+      <AccordionItem>
+        <AccordionHeader icon={<CircleRingIcon />} expandIconPosition="start">
+          Icon Start
+        </AccordionHeader>
+        <AccordionPanel>Hidden Panel</AccordionPanel>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionHeader icon={<CircleRingIcon />} expandIconPosition="end">
+          Icon End
+        </AccordionHeader>
+        <AccordionPanel>Hidden Panel</AccordionPanel>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionHeader inline icon={<CircleRingIcon />} expandIconPosition="end">
+          Icon Inline End
+        </AccordionHeader>
+        <AccordionPanel>Hidden Panel</AccordionPanel>
+      </AccordionItem>
+    </Accordion>
+  ))
+  .addStory('disabled', () => (
+    <Accordion index={[]}>

```

Comment:
```suggestion
    <Accordion>
```

Reply:
if I don't do that then by default Accordion will have the first element opened, and I want all of them closed